test-plugins/org.yakindu.sct.generator.java.test/org.yakindu.sct.generator.java.test/org/yakindu/scr/ITracing.java

```diff
@@ -0,0 +1,6 @@
+package tracingjava;
+
+public interface ITracing<T> {
+	abstract void stateEntered(T state);
+	abstract void stateExited(T state);
```

Comment:
no need for abstract keyword in an interface

Reply:
And since this is an interface we should add a little bit of documentation :)